source/plugin/api-versions.rst

```diff
@@ -0,0 +1,81 @@
+============
+API-Versions
+============
+
+This page explains which API versions exists and to which Minecraft version their implementations belong to.
+
++-------------+--------------+----------------+-------------------------------------------+
+| API-Version | Release Date | End of Support | Known Implementations (Minecraft Version) |
++=============+==============+================+===========================================+
+| *Planned*   | ---          | --             | * *SpongeForge (1.13.x)*                  |
+| *8.0.0*     |              |                | * *SpongeVanilla (1.13.x)*                |
++-------------+--------------+----------------+-------------------------------------------+
+| 7.1.0       | 2018-09-06   | Late 2019      | * SpongeForge (1.12.2)                    |
+|             |              |                | * SpongeVanilla (1.12.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+| 7.0.0       | 2018-01-01   | ^              | * SpongeForge (1.12.2)                    |
```

Comment:
I think `^` -> `Late 2019` just to be explicit.

Reply:
Do we actually support 7.0.0 until late 2019 or do we only support 7.1.0?

Reply:
I don't think we should be boxing ourselves into a time frame yet, we should perhaps leave this blank until we announce a date. Talk to @Zidane and @gabizou about this before adding this in.

We support 7.0 as it is a subset of 7.1, so any builds for 7.1 are also techncially 7.0 compatible. I wouldn't say 7.0 is end of support yet - plugins built on 7.0 will work on 7.1.

Reply:
Although there is as yet no definitive decision on support life for API 7.X, it is likely that the time-frame will be directly related to the release of a "stable" API 8 build. Only sometime thereafter will we abandon API 7.X to the wolves and hyenas. As such, any date we provide now is no more than a guess.

Reply:
I removed it for now, unless I get more specific details than last time.